frontend-react/src/hooks/UseValueSets.test.ts

```diff
@@ -20,12 +14,7 @@ import {
 describe("useValueSetsTable", () => {
     const renderWithQueryWrapper = (tableName: LookupTables) =>
         renderHook(() => useValueSetsTable<ValueSet[]>(tableName), {
-            wrapper: QueryWrapper(
-                new QueryClient({
-                    // to allow for faster testable failures
-                    defaultOptions: { queries: { retry: false } },
-                })
-            ),
+            wrapper: QueryWrapper(),
```

Comment:
Wrapper can still take a custom, but the common `testClient` has been abstracted for reuse as the default argument